### PR TITLE
add links tests

### PR DESCRIPTION
**Pull request by nehemiah-abuga**
## PR Summary

The PR is add tests to validate 2XX and #XX status response for the various links within 

Death of a loved one English and Spanish page
Disability English and Spanish page
Retirement English and Spanish page

## Related Github Issue

- task #(883)

## Type of change

- [ ] Task
- [ ] New feature - Test Automation

## Detailed Testing steps

Run ```npm run cypress:run_chrome ``` and verify all tests pass including the added links tests
Run ```npx cypress open``` and verify the links tests pass

### CODE REVIEW

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

File: cypress/e2e/ui/links.cy.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const sitePages = require("../../fixtures/site-pages.json")
+
+describe("Verify status code 200 when user navigates links", () => {
+  sitePages.forEach((sitePage) => {
+    it(`Verify success 200 response for links in ${sitePage.name} `, () => {
+      cy.visit({ url: sitePage.route })
+      cy.get("a[href]").each((link) => {
+        cy.request({
+          url: link.prop("href"),
+          failOnStatusCode: false,
```
[the remark this answers is not in the document]
Comment (nehemiah-abuga) [reply cypress/e2e/ui/links.cy.js]: I have removed failOnStatusCode. Note there are test failing because the URLs needs to be updated

**Comment by nehemiah-abuga** [comment]
@scottqueen-bixal - I have made the updates

**Review by scottqueen-bixal — APPROVED** [review]

**Comment by scottqueen-bixal** [comment]
let's hold off on putting this in upstream until this ticket is completed https://github.com/orgs/GSA/projects/48/views/10?pane=issue&itemId=30816804

just so we don't have failing tests on all the other work coming in.

File: cypress/e2e/ui/links.cy.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
+
+const sitePages = require("../../fixtures/site-pages.json")
```
Comment (scottqueen-bixal) [inline cypress/e2e/ui/links.cy.js:3]: Replace with import

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

**Review by scottqueen-bixal — APPROVED** [review]

**Comment by scottqueen-bixal** [comment]
> let's hold off on putting this in upstream until this ticket is completed https://github.com/orgs/GSA/projects/48/views/10?pane=issue&itemId=30816804
> 
> just so we don't have failing tests on all the other work coming in.

Deciding to move forward based on the recommendation that 

> ....since from QA it is done we can merge into main because the code is doing what it is supposed to do...able to catch the issues

**Pull request merged**